### ISSUE TITLE
.well-known/change-password

### ISSUE THREAD

**Issue by strarsis** [mailcow/mailcow-dockerized#3864]
## Summary
Google Chrome password manager noticed me of a compromised password.
The listed site also offers a "Change password" button - which will lead to the site domain + `./well-known/change-password`

## Motivation
The URI `well-known/change-password` is a proposed mechanism for users to easily adjust the password.

## Additional context
https://web.dev/change-password-url/

**Comment by dragoangel** [comment]
For me in case with mailcow it will strange, as users change password right on first page on first link after login. What point to add this? If you like you can simply do 301 redirect from `/.well-known/change-password` to `/` and that all.

P.s. in issue you missed `.` before `well-known` :D

**Comment by strarsis** [comment]
@dragoangel: But redirecting to `/` wouldn't display a password change page right away, right?

**Comment by dragoangel** [comment]
Yes, thats why I said "it willl strange". To get this looking nice need to have dedicated page for only password change.
